index.html

```diff
@@ -725,8 +739,85 @@ <h2 class="slogan-footer">
                     </div>
                 </div>
 
+                <div id="sutd" class="section section__sutd">
+                    <h1 class="sutd-project">SUTD Project</h1>
```

Comment:
@GracielaCesareo @lubc I don't think we should have a SUTD section in the styleguide. This should be a generic styleguide, if we start to add specific project's section it loses the purpose a little bit.

Can't we just have a "Tables" and "Labels" section? That way is explicit that we can use it in any project.

Reply:
@cleicar @arielj You have a good point. I think that refactor not require too much effort and is possible to make it before the merge